engine/jobmaster/dm/dm_jobmaster.go

```diff
@@ -201,24 +206,28 @@ func (jm *JobMaster) handleOnlineStatus(worker framework.WorkerHandle) error {
 
 // OnWorkerOffline implements JobMasterImpl.OnWorkerOffline
 func (jm *JobMaster) OnWorkerOffline(worker framework.WorkerHandle, reason error) error {
+	extBytes := worker.Status().ExtBytes
+	if len(extBytes) == 0 {
+		jm.Logger().Warn("worker status extBytes is empty", zap.String(logutil.ConstFieldWorkerKey, worker.ID()))
```

Comment:
it should not happen, otherwise we have no change to UpdateClient. Maybe we can use DPanic level and let the logger turn on `Development` for DPanic

https://github.com/pingcap/tiflow/blob/822a79f6e2e2619d00df4d6b70572cf5064b4ba3/dm/pkg/log/log.go#L118

Reply:
Let me check whether `OnWorkerOffline` can happen with receiving `OnWorkerOnline`